src/styles/index.scss

```diff
@@ -1,3 +1,108 @@
+@import 'variables';
+@import 'star';
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.card {
```

Comment:
it is better to create separate file for each BEM block and name it respectively
in your index.scss file you can keep imports and some common styles like for body or HTML tag or resetting some default styles